### PR TITLE
docs(config): add Cube Store cloud authentication configuration to Environment Variables Reference

### PR DESCRIPTION
**Pull request by hassankhan**
Story details: https://app.clubhouse.io/cube-dev/story/674

### CODE REVIEW

File: docs/content/Configuration/Environment-Variables-Reference.md [hunk 1]
```diff
@@ -125,36 +126,48 @@ databases [in this guide][link-connecting-to-db].
...
+| `CUBESTORE_AWS_SECRET_ACCESS_KEY`                   | The Secret Access Key for AWS. Required when using AWS S3                                  | [A valid AWS Secret Access Key][link-aws-creds]                               |
+| `CUBESTORE_S3_BUCKET`                               | The name of a bucket in AWS S3                                                             | -                                                                             |
+| `CUBESTORE_S3_REGION`                               | The region of a bucket in AWS S3                                                           | -                                                                             |
+| `CUBESTORE_S3_SUB_PATH`                             | The path in a AWS S3 bucket to store pre-aggregations. Optional                            | -                                                                             |
+| `CUBESTORE_GCP_SERVICE_ACCOUNT`                     | The path to a JSON key file for Google Cloud. Required when using Google Cloud Storage     | [The path to a valid Google Cloud JSON key file][link-gcp-creds-json]         |
```
Comment (ilya-biryukov) [inline docs/content/Configuration/Environment-Variables-Reference.md:158]: We could also mention that it's the same as `SERVICE_ACCOUNT` for configuring Google Cloud SDK.
Folks should be familiar with it.

**Review by ilya-biryukov — APPROVED** [review]
LGTM from my side, @keydunov might want to comment on the names of env vars.

**Comment by ilya-biryukov** [comment]
@hassankhan https://github.com/cube-js/cube.js/pull/2936 will land soon with the final variables: `CUBESTORE_GCP_CREDENTIALS` and `CUBESTORE_GCP_KEY_FILE`, semantics with the same as in cube js

**Comment by hassankhan** [comment]
@ilya-biryukov Will do, thanks for letting me know 😀

**Review by keydunov — APPROVED** [review]
LGTM. @paveltiunov / @ilya-biryukov can you confirm the correct description of the env vars?

**Comment by chidrak** [comment]
Sent from Yahoo Mail for iPhone


On Tuesday, June 15, 2021, 3:32 AM, Hassan Khan ***@***.***> wrote:




Merged #2878 into master.

**Pull request merged**